src/moped/src/main/java/core/behaviour_states/states/Manual.java

```diff
@@ -1,4 +1,22 @@
 package core.behaviour_states.states;
 
+import core.action_strategies.ActionStrategy;
+import core.action_strategies.RemoteController;
+
 public class Manual implements BehaviourState {
+
+    private ActionStrategy controller;
+
+    public Manual() {
+
+    }
+
+    public void init(ActionStrategy controller) {

```

Comment:
This init function seems weird. Shouldn't BehaviousStates themselves handle what ActionStrategys they use?

Reply:
This is for dependency injetction, just in case
We'll review after we've done the complete UML